### PR TITLE
209 improve schema tooltips

### PR DESCRIPTION
**Pull request by PaulBredl**
It can still have some refinement, e.g. the "anyOf", "oneOf" and "allOf" description look weird, but I think for now I should not invest way more time in it.

image

### CODE REVIEW

File: frontend/config-assistant/src/components/gui-editor/GuiEditorPanelJsonSchema.vue [hunk 1]
```diff
@@ -47,13 +47,14 @@ function zoomIntoPath(pathToAdd: Path) {
       :root-name="sessionStore.fileSchema?.title ?? 'root'"
       :path="sessionStore.currentPath"
       @update:path="newPath => updatePath(newPath)" />
-    <PropertiesPanel
```
Comment (PaulBredl) [inline frontend/config-assistant/src/components/gui-editor/GuiEditorPanelJsonSchema.vue:50]: Applying classes on components does no longer work apparantly

File: frontend/config-assistant/src/components/gui-editor/PropertiesPanel.vue [hunk 2]
```diff
@@ -267,7 +301,8 @@ function expandElement(node: any) {
             class="w-full"
             :nodeData="slotProps.node.data"
             @update_property_value="updateData"
-            bodyClass="w-full" />
+            bodyClass="w-full"
+            @keydown.ctrl.i="event => showInfoOverlayPanelInstantly(slotProps.node.data, event)" />
```
Comment (PaulBredl) [inline frontend/config-assistant/src/components/gui-editor/PropertiesPanel.vue:305]: With keyboard controls, you can open the  overlay by pressing "ctrl + i"

File: frontend/config-assistant/src/helpers/schema/schemaDescriptor.ts
Comment (PaulBredl) [inline frontend/config-assistant/src/helpers/schema/schemaDescriptor.ts:1]: Main contribution of this PR

Comment (PaulBredl) [inline frontend/config-assistant/src/helpers/schema/schemaDescriptor.ts:1]: Main contribution of this PR

Comment (Logende) [reply frontend/config-assistant/src/helpers/schema/schemaDescriptor.ts:1]: Wow, impressive

**Review by Logende — APPROVED** [review]

**Pull request merged**